scripts/functional-tests.sh

```diff
@@ -8,6 +15,11 @@ export SCRIPTDIR=$(dirname "$0")
 # build a prompt string that includes the time, source file, line number, and function name
 export PS4='+$(date +"%Y-%m-%d %T") ${BASH_VERSION}:${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
+# Support overriding the test directory for local testing otherwise creates a temporary directory
+TEST_DIR=${TEST_DIR:-$(mktemp -d --tmpdir=/var/tmp instructlab-test-tmp.XXXXXX)}
+# Support cleaning up the test directory once the script is done, defaults to True, set any value to disable
+export SKIP_TEST_DIR_CLEANUP
```

Comment:
minor nit: strictly speaking, empty value is also "any value". This should probably say that any *non-empty* value should be set; also, `export`s aren't meaningful in this context, so if you'd like to keep it for documentation purposes, just do `SKIP_TEST_DIR_CLEANUP=`.